lib/index.js

```diff
@@ -85,25 +95,96 @@ Vero.prototype.track = function(payload, fn) {
  * @api public
  */
 
-Vero.prototype.alias = request('/users/reidentify', 'put');
+Vero.prototype.alias = request('alias', '/users/reidentify', 'put');
+
+/**
+ * Add or Remove Tags.
+ *
+ * http://developers.getvero.com/?bash#tags
+ *
+ * @param {Facade} msg
+ * @param {Object} tags
+ * @param {Function} fn
+ * @api public
+ */
+
+Vero.prototype.addOrRemoveTags = function(msg, tags, fn) {
+  // Check to ensure tags object is properly formatted.
+  var errmsg;
+  if (Object.prototype.toString.call(tags) !== '[object Object]') {
+    errmsg = 'Tag option must be passed as an Object.';
+    return fn(new Error(errmsg));
+  }
+
+  if (!tags.action || tags.action !== 'add' && tags.action !== 'remove') {
+    errmsg = 'Tags action value must be either \'add\' or \'remove\'';
+    return fn(new Error(errmsg));
+  }
+
+  if (!tags.tags || !Array.isArray(tags.tags)) {
+    errmsg = 'Could not detect tag values. Tag values must be passed as array of strings.';
```

Comment:
If not, would make the messaging more clear. Is tag values and tag options different? Might be better to describe this message in Segment language

`"Must send `properties.tags` as an object..."`
`"Must send `properties.tags.value as an array..."`